nixos/modules/services/networking/adguardhome.nix

```
@@ -89,6 +89,9 @@ in
         On start and if {option}`mutableSettings` is `true`,
         these options are merged into the configuration file on start, taking
         precedence over configuration changes made on the web interface.
+
+        Set this to `null` (default) for a non-declarative configuration without any
+        nix-supplied default values.
```

Would "values" be more clear, since it wouldn't *just* be default values if other values are added? Not sure, what do you think?

```suggestion
        Set this to `null` (default) for a non-declarative configuration without any
        Nix-supplied values.
```

---

I think that your suggestion is more clear, but I think it would make sense to make it explicit that attrsets have default values.


What do you think about this?
```suggestion
        Set this to `null` (default) for a non-declarative configuration without any
        Nix-supplied values. 
        Attribute sets are supplied with a default `schema_version`, `bind_host` and `bind_port`.
```

---

How about "declarative configurations are supplied with a default `schema_version`, `bind_host`, and `bind_port`"?

---

```suggestion
        Set this to `null` (default) for a non-declarative configuration without any
        Nix-supplied values.
        Declarative configurations are supplied with a default `schema_version`, `bind_host`, and `bind_port`.
```

Looks fine for me, too. The comma in front of the and is common in US English, but not in British English (This is just a note for myself as I had to look that up).

---

That works.